integration-libs/cdc/manage-users/disable-edit/cdc-user-details.module.ts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Rename the enclosing folder from `disable-edit` to `user-details`

Reply:
done